homeassistant/components/airzone_cloud/entity.py

```diff
@@ -22,6 +28,37 @@ def get_airzone_value(self, key: str) -> Any:
         """Return Airzone Cloud entity value by key."""
 
 
+class AirzoneAidooEntity(AirzoneEntity):
+    """Define an Airzone Cloud Aidoo entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        entry: ConfigEntry,
+        aidoo_id: str,
+        aidoo_data: dict[str, Any],
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self.aidoo_id = aidoo_id
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, f"{entry.unique_id}_{aidoo_id}")},
+            manufacturer=MANUFACTURER,
+            name=aidoo_data[AZD_NAME],
+            via_device=(DOMAIN, f"{entry.unique_id}_{aidoo_data[AZD_WEBSERVER]}"),
+        )
+
+    def get_airzone_value(self, key: str) -> Any:
+        """Return Aidoo value by key."""
+        value = None
+        if aidoo := self.coordinator.data[AZD_AIDOOS].get(self.aidoo_id):
+            if key in aidoo:
+                value = aidoo[key]
+        return value
```

Comment:
```suggestion
        if aidoo := self.coordinator.data[AZD_AIDOOS].get(self.aidoo_id):
            return aidoo.get(key)
        return None
```

You could do the above if there isn't a difference between `aidoo[key]` being non-existent and `None` and it saves one dict lookup

Reply:
Yeah, the only disadvantage is the code coverage, since line 57 won't be tested.

Reply:
Done in 0b76625265274c4e92998677ca4bce02dcd74f53 with a minor modification to keep 100% coverage.

Reply:
You could probably also write it as 

`return (aidoo := self.coordinator.data[AZD_AIDOOS].get(self.aidoo_id)) and aidoo.get(key)`

Reply:
I'm assuming the data looks like this

`dict = {"key": {"subkey": "value"}}`

which effectively makes it

`return (subdict := dict.get("key")) and subdict.get("subkey")`